app/models/follow.rb

```diff
@@ -0,0 +1,31 @@
+class Follow < ActiveRecord::Base
+  belongs_to :user
```

Comment:
I think a cool improvement here (even if there is no actual use on the code right now) would have been to use a [counter_cache](http://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache)

Something like `belongs_to :user, counter_cache: :following_count`

Without forgetting the migration:
```
    add_column :users, :following_count, :integer, default: 0
    add_index :users, :following_count
```